components/molecules/InsightRow/insight-row.tsx

```diff
@@ -77,14 +77,13 @@ const InsightRow = ({ insight, user }: InsightRowProps) => {
               </div>
             </div>
             <div className="flex-1 hidden md:flex  justify-end">
-              {(user?.user_metadata.sub && Number(user?.user_metadata.sub) === Number(insight.user_id)) ||
-                (!insight.is_featured && (
-                  <Link href={`/hub/insights/${insight.id}/edit`}>
-                    <span className=" bg-light-slate-1 inline-block rounded-lg p-2.5 border mr-2 cursor-pointer">
-                      <BsPencilFill title="Edit Insight Page" className="text-light-slate-10 text-lg" />
-                    </span>
-                  </Link>
-                ))}
+              {!insight.is_featured && (
+                <Link href={`/hub/insights/${insight.id}/edit`}>
+                  <span className=" bg-light-slate-1 inline-block rounded-lg p-2.5 border mr-2 cursor-pointer">
+                    <BsPencilFill title="Edit Insight Page" className="text-light-slate-10 text-lg" />
+                  </span>
+                </Link>
+              )}
```

Comment:
@open-sauced/engineering - I could use some suggestions here:

this chunk before was checking if the passed in user is the user of the insight page using `user_id`: I thought I'd apply the same treatment here and use `getInsightTeamMember` but wasn't sure the best way to go about that or if there were better ideas since this seems to be a more deeply embedded react component.

For now, it only returns the higlighted insights so it'll need updating.

Reply:
We may need a separate endpoint for this to get the insights they have access to edit, and display the editing icon based on that list.

Reply:
Yeah, I think you're right: there's no great way to get the membership status for all insights from an authenticated user. Opened this and will tackle that before moving forward with this: https://github.com/open-sauced/api/issues/449

Reply:
This has now been addressed in the latest push along side changes from: https://github.com/open-sauced/api/pull/448